polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -353,8 +314,8 @@ class PythonInterpreter(val symbolTable: RuntimeSymbolTable) extends LanguageInt
 object PythonInterpreter {
   class Factory extends LanguageInterpreter.Factory[IO] {
     override val languageName: String = "Python"
-    override def apply(dependencies: List[(String, File)], symbolTable: RuntimeSymbolTable): PythonInterpreter =
-      new PythonInterpreter(symbolTable)
+    override def apply(dependencies: List[(String, File)], kernelContext: KernelContext): PythonInterpreter =
```

Comment:
Oh yeah, factories now take `KernelContext` instead of `RuntimeSymbolTable` (since the latter is gone)